src/Aws/Lambda/Runtime.hs

```diff
@@ -29,84 +18,98 @@ import qualified Aws.Lambda.Runtime.Environment as Environment
 import qualified Aws.Lambda.Runtime.Error as Error
 import qualified Aws.Lambda.Runtime.Publish as Publish
 import qualified Control.Exception as Unchecked
+import Control.Exception.Safe.Checked (Throws, catch, throw)
+import qualified Control.Exception.Safe.Checked as Checked
+import Control.Monad (forever)
+import Data.Aeson (ToJSON (toJSON))
+import Data.IORef (newIORef)
+import Data.Text (Text, unpack)
+import qualified Network.HTTP.Client as Http
+import System.IO (hFlush, stderr, stdout)
 
 -- | Runs the user @haskell_lambda@ executable and posts back the
 -- results. This is called from the layer's @main@ function.
-runLambda :: forall context. IO context -> Runtime.RunCallback context -> IO ()
+runLambda :: forall context t. IO context -> Runtime.RunCallback t context -> IO ()
```

Comment:
What is `t` in this case? Could you give it a more semantic name please?

Reply:
Consider the `t` eliminated. :neckbeard: